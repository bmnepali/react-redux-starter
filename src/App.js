import React from 'react';
import { BrowserRouter } from 'react-router-dom';

// Lazy load routes
const Routes = React.lazy(() => import('./Routes'));

const App = () => (
  <BrowserRouter>
    <React.Suspense>
      <Routes />
    </React.Suspense>
  </BrowserRouter>
);

export default App;
