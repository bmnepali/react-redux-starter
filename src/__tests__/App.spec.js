import * as React from 'react';

// Load Component
import App from '../App';

describe('App Component', () => {
  describe('render testing', () => {
    it('renders App component without any error', () => {
      shallow(<App />);
    });

    it('matches the snapshot', () => {
      const wrapper = shallow(<App />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
