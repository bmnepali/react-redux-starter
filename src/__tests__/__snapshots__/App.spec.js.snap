// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component render testing matches the snapshot 1`] = `
<BrowserRouter>
  <Suspense>
    <lazy />
  </Suspense>
</BrowserRouter>
`;
